src/FunctionApps/python/tests/GenerateCSVs/test_patient.py

```diff
@@ -0,0 +1,95 @@
+from GenerateCSVs.patient import parse_patient_resource
+
+
+def test_parse_patient_resource():
+
+    patient = {
+        "fullUrl": "urn:uuid:c53f9ad8-34c1-ce05-c0f6-7a0ea7bd8483",
```

Comment:
Tests look great - like this is totally fine, but if you want to load them from files (like `tests/IntakePipeline/assets/patient_bundle.json`) that'd be cool too